app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -144,7 +144,8 @@
           </FrameLayout>
         </LinearLayout>
       </LinearLayout>
-
+      <!-- FrameLayout is used here instead of LinearLayout to properly adjust the visible thickness
```

Comment:
```suggestion
      <!-- FrameLayout is used here instead of LinearLayout to properly adjust the spacing
```

Along with my suggestion below, may be worth moving words up/down per line-wrapping.